src/S3/SignatureTraitS3.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Aws\S3;
+
+use Aws\Signature\SignatureTrait;
+
+/**
+ * Provides signature calculation for SignatureV4.
+ */
+trait SignatureTraitS3
+{
+    use  SignatureTrait;
```

Comment:
I'm not sure what the point of this trait is. Couldn't classes just use `Aws\Signature\SignatureTrait` directly?